projects/1.19/assets/heat-and-climate/dcs_climate/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+  "block.dcs_climate.food.cropblock_apium_common":"茴香",
+  "block.dcs_climate.food.cropblock_apium_rare":"防风草",
+  "block.dcs_climate.food.cropblock_brassica_wild":"油菜",
+  "block.dcs_climate.food.cropblock_brassica_common":"萝卜叶",
+  "block.dcs_climate.food.cropblock_brassica_rare":"卷心菜",
+  "block.dcs_climate.food.cropblock_brassica_epic":"萝卜",
+  "block.dcs_climate.food.cropblock_capsicum_wild":"辣椒",
+  "block.dcs_climate.food.cropblock_capsicum_common":"甜椒",
+  "block.dcs_climate.food.cropblock_capsicum_rare":"彩椒",
+  "block.dcs_climate.food.cropblock_cereals_wild":"燕麦",
+  "block.dcs_climate.food.cropblock_cereals_common":"黑麦",
+  "block.dcs_climate.food.cropblock_cereals_rare":"大麦",
+  "block.dcs_climate.food.cropblock_ginger_wild":"生姜",
+  "block.dcs_climate.food.cropblock_ginger_common":"豆蔻",
+  "block.dcs_climate.food.cropblock_ginger_rare":"姜黄",
+  "block.dcs_climate.food.cropblock_herb_wild":"薄荷",
+  "block.dcs_climate.food.cropblock_herb_common":"罗勒",
+  "block.dcs_climate.food.cropblock_herb_rare":"紫苏",
+  "block.dcs_climate.food.cropblock_herb_epic":"薰衣草",
+  "block.dcs_climate.food.cropblock_knotweed_wild":"酢浆草",
+  "block.dcs_climate.food.cropblock_knotweed_common":"荞麦",
+  "block.dcs_climate.food.cropblock_knotweed_rare":"蝶豆花",
+  "block.dcs_climate.food.cropblock_mallow_wild":"黄麻",
+  "block.dcs_climate.food.cropblock_mallow_common":"棉花",
+  "block.dcs_climate.food.cropblock_mallow_rare":"紫罗兰",
+  "block.dcs_climate.food.cropblock_morningglory_wild":"田旋花",
+  "block.dcs_climate.food.cropblock_morningglory_common":"空心菜",
+  "block.dcs_climate.food.cropblock_morningglory_rare":"红薯",
+  "block.dcs_climate.food.cropblock_morningglory_epic":"牵牛花",
+  "block.dcs_climate.food.cropblock_pedalia_wild":"奇亚籽",
+  "block.dcs_climate.food.cropblock_pedalia_common":"芝麻",
+  "block.dcs_climate.food.cropblock_pedalia_rare":"南非钩麻'",
+  "block.dcs_climate.food.cropblock_peas_wild":"豌豆",
+  "block.dcs_climate.food.cropblock_peas_common":"鹰嘴豆",
+  "block.dcs_climate.food.cropblock_peas_rare":"大豆",
+  "block.dcs_climate.food.cropblock_peas_epic":"红豆",
+  "block.dcs_climate.food.cropblock_reed_wild":"芦苇",
+  "block.dcs_climate.food.cropblock_reed_common":"高粱",
+  "block.dcs_climate.food.cropblock_reed_rare":"玉米",
+  "block.dcs_climate.food.cropblock_rice_wild":"茭白",
+  "block.dcs_climate.food.cropblock_rice_common":"大米",
+  "block.dcs_climate.food.cropblock_rice_rare":"香米",
+  "block.dcs_climate.food.cropblock_solanum_wild":"颠茄",
+  "block.dcs_climate.food.cropblock_solanum_common":"茄子",
+  "block.dcs_climate.food.cropblock_solanum_rare":"番茄",
+  "block.dcs_climate.food.cropblock_solanum_epic":"灯笼果",
+  "block.dcs_climate.food.cropblock_palm_wild":"椰子",
+  "block.dcs_climate.food.cropblock_palm_common":"椰枣",
+  "block.dcs_climate.food.cropblock_palm_rare":"油棕",
+
+  "block.dcs_climate.food.sapling_beech_wild":"山毛榉树苗",
+  "block.dcs_climate.food.sapling_beech_common":"核桃树苗",
+  "block.dcs_climate.food.sapling_beech_rare":"白橡木树苗",

```

```suggestion
  "block.dcs_climate.food.sapling_beech_rare":"白橡树苗",
```
```suggestion
  "block.dcs_climate.food.sapling_beech_rare":"白橡树树苗",
```
见原版深色橡木/深色橡树的处理，不过群里讨论过了所以`白橡`作为种名，或者`白橡树`作为种名都可以